src/genie/libs/parser/asa/show_access_list.py

```diff
@@ -0,0 +1,187 @@
+''' show_access_list.py
+
+Parser for the following show commands:
+    * show access-list
+'''
+
+# Python
+import re
+import logging
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Optional
+
+# =============================================
+# Schema for 'show access-list'
+# =============================================
+class ShowAccessListSchema(MetaParser):
+    """Schema for
+        * show access-list
+    """
+
+    schema = {
+    	'access-list': {
+            Any(): {
+                Optional('elements'): int,
+#               'name_hash': str,
+                'entry': {
+                    Any(): {
+                      Optional('action'): str, 
+                      Optional('protocol'): str,
+                      Optional('source'): {
+                          Optional('host'): str,
+                          Optional('object_group'): str,
+                          Optional('any'): str,
+                          Optional('port'): str,
+                      },
+                      Optional('destination'): {
+                          Optional('host'): str,
+                          Optional('object_group'): str,
+                          Optional('any'): str,
+                          Optional('port'): str,
+                      },
+                      Optional('log'): bool,
+                      Optional('disable'): bool,
+                      Optional('hitcnt'): int,
+                      Optional('acl_hash'): str,
+                      Optional('remark'): str,
+                    }
+                },
+            },
+        },
+        Optional('cached_acl_log_flows'): {
+            'total': int,
+            'denied': int,
+            'deny_flow_max': int,
+        },
+        Optional('alert_interval'): int,
+    }
+
+# =============================================
+# Parser for 'show access-list'
+# =============================================
+class ShowAccessList(ShowAccessListSchema):
+    """Parser for
+        * show access-list
+    """
+
+    cli_command = 'show access-list'
+
+    def cli(self, output=None):
+        if output is None:
+            # excute command to get output
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        ret_dict = {}
+        ret_dict['access-list'] = {}
```

Comment:
Please remove this. We only want to add to the dictionary within the for-loop. This way if nothing is parsed an empty dictionary will get returned (we expect this behaviour)